docs/clustering/logical-replication-setup.rst

```diff
@@ -0,0 +1,146 @@
+.. _logical_replication_setup:
+
+==================================================
+Logical replication setup between CrateDB clusters
+==================================================
+
+Logical replication is a method of data replication between multiple clusters.
+As a publish/subscribe model, it allows a publishing cluster to make certain
+tables available for subscription. Subscribing clusters subscribe to a
+publication to replay any changes to the data on their side.
```

Comment:
The last sentence is a bit complicate IMO. Maybe "Any change on a published table will be replicated to the matching table in the subscribing cluster"?